### ISSUE TITLE
Credo.Check.Consistency.MultiAliasImportRequireUse.Multi misreporting for modules in functions

### ISSUE THREAD

**Issue by KronicDeth** [rrrene/credo#309]
# Environment

* Credo version (`mix credo -v`): `0.6.0`
* Erlang/Elixir version (`elixir -v`): `19.2` / `1.3.4`
* Operating system: OSX 

### What were you trying to do?

```
 defp disconnect_during_prefetch_count(context) do
    KillOnce.start_link()

    defmodule DisconnectDuringPrefetchCount do
      @moduledoc false

      use Retort.AMQP

      @spec prefetch_count(%AMQP.Channel{}, non_neg_integer) :: :ok
      def prefetch_count(channel = %AMQP.Channel{conn: %AMQP.Connection{pid: pid}}, prefetch_count) do
        KillOnce.kill_once(pid)

        super(channel, prefetch_count)
      end
    end

    %{amqp: __MODULE__.DisconnectDuringPrefetchCount}
  end
```

### Expected outcome

No error reported on line 322 (`use Retort.AMQP`).

### Actual outcome

```
Consistency                                                                   
┃ 
┃ [C] ↗ Most of the time you are using the multi-alias/require/import/use 
┃       syntax, but here you are using multiple single directives
┃       test/retort/channel_test.exs:322 (Retort.ChannelTest.disconnect_during_prefetch_count.DisconnectDuringPrefetchCount)
```

Weirdly, `use Retort.AMQP` is used in all the setup callback defined modules (https://github.com/C-S-D/retort/blob/master/test/retort/channel_test.exs#L201-L325) without problems except for the last one, which is on line 315 of master, but 322 of the branch that upgrades to credo `0.6.0` as reported above.

**Comment by rrrene** [comment]
Thx Luke! We'll have to look into it. :+1:

**Comment by Eiji7** [comment]
@rrrene: I have a problem related to that check. I tried to comment this one, but it does not work.
My `config/.credo.exs`:
```elixir
%{
  configs: [
    %{
      check_for_updates: true,
      checks: [
        {Credo.Check.Consistency.ExceptionNames},
        {Credo.Check.Consistency.LineEndings},
        # {Credo.Check.Consistency.MultiAliasImportRequireUse},
        {Credo.Check.Consistency.ParameterPatternMatching},
        {Credo.Check.Consistency.SpaceAroundOperators},
        {Credo.Check.Consistency.SpaceInParentheses},
        {Credo.Check.Consistency.TabsOrSpaces},
        {Credo.Check.Design.AliasUsage, priority: :higher},
        {Credo.Check.Design.DuplicatedCode, excluded_macros: []},
        {Credo.Check.Design.TagFIXME},
        {Credo.Check.Design.TagTODO, exit_status: 2},
        {Credo.Check.Readability.FunctionNames},
        {Credo.Check.Readability.LargeNumbers},
        {Credo.Check.Readability.MaxLineLength, priority: :higher, max_length: 100},
        {Credo.Check.Readability.ModuleAttributeNames},
        {Credo.Check.Readability.ModuleDoc},
        {Credo.Check.Readability.ModuleNames},
        {Credo.Check.Readability.NoParenthesesWhenZeroArity},
        {Credo.Check.Readability.ParenthesesInCondition},
        {Credo.Check.Readability.PredicateFunctionNames},
        {Credo.Check.Readability.PreferImplicitTry},
        {Credo.Check.Readability.RedundantBlankLines},
        {Credo.Check.Readability.Specs},
        {Credo.Check.Readability.StringSigils},
        {Credo.Check.Readability.TrailingBlankLine},
        {Credo.Check.Readability.TrailingWhiteSpace},
        {Credo.Check.Readability.VariableNames},
        {Credo.Check.Refactor.ABCSize},
        {Credo.Check.Refactor.CondStatements},
        {Credo.Check.Refactor.CyclomaticComplexity},
        {Credo.Check.Refactor.DoubleBooleanNegation},
        {Credo.Check.Refactor.FunctionArity},
        {Credo.Check.Refactor.MatchInCondition},
        {Credo.Check.Refactor.NegatedConditionsInUnless},
        {Credo.Check.Refactor.NegatedConditionsWithElse},
        {Credo.Check.Refactor.Nesting},
        {Credo.Check.Refactor.PipeChainStart},
        {Credo.Check.Refactor.UnlessWithElse},
        {Credo.Check.Refactor.VariableRebinding},
        {Credo.Check.Warning.BoolOperationOnSameValues},
        {Credo.Check.Warning.IExPry},
        {Credo.Check.Warning.IoInspect},
        {Credo.Check.Warning.NameRedeclarationByAssignment},
        {Credo.Check.Warning.NameRedeclarationByCase},
        {Credo.Check.Warning.NameRedeclarationByDef},
        {Credo.Check.Warning.NameRedeclarationByFn},
        {Credo.Check.Warning.OperationOnSameValues},
        {Credo.Check.Warning.OperationWithConstantResult},
        {Credo.Check.Warning.UnusedEnumOperation},
        {Credo.Check.Warning.UnusedFileOperation},
        {Credo.Check.Warning.UnusedKeywordOperation},
        {Credo.Check.Warning.UnusedListOperation},
        {Credo.Check.Warning.UnusedPathOperation},
        {Credo.Check.Warning.UnusedRegexOperation},
        {Credo.Check.Warning.UnusedStringOperation},
        {Credo.Check.Warning.UnusedTupleOperation},
      ],
      color: true,
      files: %{
        excluded: [~r"/_build/", ~r"/deps/"],
        included: ["lib/", "src/", "web/"],
      },
      name: "default",
      requires: [],
      strict: true,
    }
  ]
}
```
For example changing: `strict` to `false` works, so file is visible for it.
I tried to remove: `_build`, `deps/credo` and `mix.lock`. After execute: `mix deps.get` and `mix credo` I still have this messages.
Is it related to this issue or it's just somehow cached?

**Comment by rrrene** [comment]
@Eiji7 You disable a check by setting the second tuple element to `false`:

```elixir
      #
      # You can customize the parameters of any check by adding a second element
      # to the tuple.
      #
      # To disable a check put `false` as second element:
      #
      #     {Credo.Check.Design.DuplicatedCode, false}
      #
      checks: [
      ]
```

**Comment by rrrene** [comment]
I decided to disable the `MultiAliasImportRequireUse` check by default. It is not up to speed (yet) and in need of a big refactoring/re-implementation.

Everybody, thx for your reports! Community contribution is what makes this project breathe :+1:
